lib/open_food_network/scope_variant_to_hub.rb

```diff
@@ -41,7 +41,11 @@ def on_demand
       #   - updates variant_override.count_on_hand
       #   - does not create stock_movement
       #   - does not update stock_item.count_on_hand
+      # If it is a variant override with on_demand:
+      #   - don't change stock or call super (super would change the variant's stock)
       def move(quantity, originator = nil)
+        return if @variant_override.andand.on_demand
```

Comment:
I think it's worth unit testing as well because someday (I hope) we'll want to refactor this

Reply:
I got a lot of errors trying to unit test this, as I think there needs to be line items and stock locations set up for it to work.

Reply:
Done